### PR TITLE
Dockerized

### PR DESCRIPTION
**Pull request by codearranger**
## Checklist

- [x] Read CONTRIBUTING.md, and accept the CLA by including the provided snippet. We will not accept PR without this.
```
I, @codearranger, confirm that I have read and understood the terms of the CLA of Kyutai-labs, as outlined in the repository's CONTRIBUTING.md, and I agree to be bound by these terms.
```
- [x] Run pre-commit hook.
```
$ pre-commit run --all-files
flake8 on moshi package..................................................Passed
pyright on moshi package.................................................Passed
flake8 on moshi_mlx package..............................................Passed
pyright on moshi_mlx package.............................................Passed
```
- [ ] If you changed Rust code, run `cargo check`, `cargo clippy`, `cargo test`.

## PR Description

Added Dockerfiles and docker compose to run the service on a Linux with the NVIDIA Container Toolkit

#### Install the NVIDIA Container Toolkit

https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html

#### Start the docker container
```docker compose up```

### CODE REVIEW

**Comment by adefossez** [comment]
Could you add a section in the readme explaining how to use the docker image? Then I can accept the PR.
Thanks a lot!

**Comment by codearranger** [comment]
@adefossez done

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+version: "3.8"
```
[the remark this answers is not in the document]
Comment (codearranger) [reply docker-compose.yml:1]: It looks like it's been removed and it's ignored now.

https://docs.docker.com/reference/compose-file/version-and-name/

**Comment by adefossez** [comment]
Thanks a lot, merged!

**Pull request merged**